.ci/deps.opam.sh

```diff
@@ -1,13 +1,14 @@
 set -e
 set -x
 
-TERM=dumb
+export TERM=dumb
```

Comment:
As i mentioned in the last PR, this is a general bug and should have separate issue describing problem.

Specifically, why is it important to set it only when doing opam and not for other scripts.

Reply:
Here is what happens, if I don't use export...
I will try to get a better insight into this problem :smile: 
https://circleci.com/gh/harshhx17/coala-bears/445

Reply:
This error was also reported here
https://github.com/ocaml/opam/issues/2978

Reply:
I don't understand why it was passing in the circleci 1.0
Should I create a seperate pr for this...
@jayvdb

Reply:
I think I said in the last PR that the Circle CI set TERM for us.

Use git blame to find out more about these lines, and how might we improve it.
```
$ git grep TERM .ci
.ci/deps.apt.sh:TERM=dumb
.ci/deps.opam.sh:TERM=dumb
.ci/deps.pip.sh:TERM=dumb
.ci/deps.sh:TERM=dumb
```

Reply:
Thanks!
I will try look into it 🙂

Reply:
The first pr(https://github.com/coala/coala-bears/pull/1339) related to https://github.com/coala/coala-bears/issues/1123 did not solve the issue for circle...
While the second pr(https://github.com/coala/coala-bears/pull/1705), is not merged and files have changed a lot since then(for instance deps.sh was split into smaller files)